addons/udes_stock_refactoring/tests/test_refactoring.py

```diff
@@ -508,6 +508,29 @@ def test_2_pallets_over_backorder_preserves_package_levels_in_picking_chain(self
             picking._action_done()
             self._assert_package_levels(goods_out_pickings)
 
+    def test_refactors_partially_unavailable_stock_into_new_move(self):
+        """The system should split moves into available and unavailable."""
+        products_info = [
+            {"product": self.apple, "uom_qty": 15},
+        ]
+        moves = self.create_move(self.picking, products_info)
+        original_move = moves.filtered(lambda m: m.product_id == self.apple)
```

Comment:
Nit: I believe dont need original_move by filtering moves with product apple.
From what i see moves = original_move so in the asserts below can use directly moves and refactored moves.

Reply:
Good point - changed, please review.